### PR TITLE
Feat/adding argocd repository project scoped

### PR DESCRIPTION
**Pull request by felipem1210**

### CODE REVIEW

File: scripts/testacc_prepare_env.sh [hunk 1]
```diff
@@ -19,11 +19,11 @@ if [[ -z "${ARGOCD_CI}" ]]; then
   kind load docker-image redis:6.2.4-alpine --name argocd
   kind load docker-image ghcr.io/dexidp/dex:v2.27.0 --name argocd
   kind load docker-image alpine:3 --name argocd
-  kind load docker-image quay.io/argoproj/argocd:${ARGOCD_VERSION:-v1.8.7} --name argocd
+  kind load docker-image quay.io/argoproj/argocd:${ARGOCD_VERSION:-v2.4.0-rc3} --name argocd
 fi
 
-echo "\n--- Install ArgoCD ${ARGOCD_VERSION:-v1.8.7}\n"
-curl https://raw.githubusercontent.com/argoproj/argo-cd/${ARGOCD_VERSION:-v1.8.7}/manifests/install.yaml > manifests/install/argocd.yml &&
+echo "\n--- Install ArgoCD ${ARGOCD_VERSION:-v2.4.0-rc3}\n"
```
Comment (oboukili) [inline scripts/testacc_prepare_env.sh:25]: Let's not use release candidate by default for testing please

**Review by oboukili — CHANGES_REQUESTED** [review]
Thanks! This PR is missing acceptance tests, could you add them?

**Comment by felipem1210** [comment]
Hi @oboukili , I checked the test code for projects and repositories and I am not sure how can I create the project token and after that use that token to test the creation of a project scoped repo using that token. Honestly, I'm really really noob in Go programming so I will take my time trying to create that test :D (unless somebody want to give a hand)

**Comment by felipem1210** [comment]
Will try to do it next week

**Pull request closed without merging**